app/views/works/new_import.html.erb

```diff
@@ -59,7 +59,7 @@
         </ul>
       </dd>
       <% if current_user.archivist %>
-        <dt><%= label_tag ts("Archiving for others") %></dt>
+        <dt><%= label_tag "importing_for_other", ts("Archiving for others") %></dt>
```

Comment:
This doesn't seem to correspond to the id of a [labelable form element](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Form_labelable), so I don't think it'll actually accomplish anything, but it won't hurt anything either, since right now the for attribute is just "Archiving_for_others" (which also does not correspond to anything).

Reply:
I don't remember now why I thought that was a good idea. I suspect RubyMine or the Safari dev tools told me that was missing the attribute - I probably saw the adjacent `importing_for_others` checkbox (which already has a label) and then typoed it to boot. I'll remove it because what's the point?